.github/workflows/publish-version.yml

```diff
@@ -143,25 +64,30 @@ jobs:
         env:
           STX_NETWORK: testnet
 
-      - name: Install `app/` packages
-        run: cd app && yarn --frozen-lockfile --ignore-scripts
-        env:
-          STX_NETWORK: testnet
-
       - name: Install `concat-map` - Windows only
         run: yarn add concat-map --ignore-scripts --frozen-lockfile
         if: matrix.os == 'windows-latest'
         env:
           STX_NETWORK: testnet
 
+      - name: Install `app/` packages
+        run: cd app && yarn --frozen-lockfile --ignore-scripts
+        env:
+          STX_NETWORK: testnet
```

Comment:
I see the `STX_NETWORK` env var being set in many jobs. Can we just set the env var once globally at the top of the workflow? It would be more visible and easier to change when the time comes.

Reply:
Nice, didn't know you could do this